spec/std/exception_spec.cr

```diff
@@ -0,0 +1,27 @@
+require "spec"
+
+# See https://github.com/crystal-lang/crystal/pull/3194#issuecomment-242474441
+class Exception
+  def inspect_with_backtrace(io : IO)
+    io << message << " (" << self.class << ")\n"
+    backtrace?.try &.each do |frame|
```

Comment:
Definitely. @Sija please add the `?` `exception.cr` and remove the Exception class in `exception_spec.cr`, otherwise the code that is been tested is not the actual code. I made the shared snippet to illustrate and quick check if the error was the missing `?`.